packages/core/src/sass/app/_common/components/account-common.scss

```diff
@@ -98,7 +98,7 @@
         &__fieldset {
             .dc-autocomplete,
             .dc-select-native {
-                margin-bottom: 3.2rem;
+                // margin-bottom: 3.2rem;
```

Comment:
lets remove this commented code

Reply:
Updated the code and I re-enabled this code.